lib/privatekey.js

```diff
@@ -157,14 +157,18 @@ PrivateKey._transformBuffer = function(buf, network) {
   }
 
   info.network = Networks.get(buf[0], 'privatekey');
```

Comment:
Yep, line 160 to 164 wasn't necessary from before since `info.network = Networks.get(buf[0], 'privatekey');` is all that is needed.